Code/FindRGL.cmake

```diff
@@ -11,29 +11,32 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-set(RGL_VERSION 0.15.0)
+set(RGL_VERSION 0.18.0)
 set(RGL_TAG v${RGL_VERSION})
 
-set(RGL_LINUX_ZIP_URL https://github.com/RobotecAI/RobotecGPULidar/releases/download/${RGL_TAG}/Linux-x64.zip)
+set(RGL_LINUX_ZIP_FILENAME_BASE RGL-full-linux-x64)
+set(RGL_LINUX_ZIP_FILENAME ${RGL_LINUX_ZIP_FILENAME_BASE}.zip)
+
+set(RGL_LINUX_ZIP_URL https://github.com/RobotecAI/RobotecGPULidar/releases/download/${RGL_TAG}/${RGL_LINUX_ZIP_FILENAME})
 set(RGL_SRC_ZIP_URL https://github.com/RobotecAI/RobotecGPULidar/archive/refs/tags/${RGL_TAG}.zip)
```

Comment:
Instead of downloading the source of the whole project, you can download the desired API header files only:
- [https://raw.githubusercontent.com/RobotecAI/RobotecGPULidar/${RGL_TAG}/include/rgl/api/core.h](https://raw.githubusercontent.com/RobotecAI/RobotecGPULidar/v0.18.0/include/rgl/api/core.h)
- [https://raw.githubusercontent.com/RobotecAI/RobotecGPULidar/${RGL_TAG}/extensions/ros2/include/rgl/api/extensions/ros2.h](https://raw.githubusercontent.com/RobotecAI/RobotecGPULidar/v0.18.0/extensions/ros2/include/rgl/api/extensions/ros2.h)

Reply:
I don't see the reason for this since the source code download time is miniscule.

Reply:
Especially since this is the more future proof way and It's been handled this way up to this point.

Reply:
Okay, I've looked into it and this would simplify the code a bit since there wouldn't be any archive extraction.